torch/_functorch/_aot_autograd/logging_utils.py

```diff
@@ -0,0 +1,131 @@
+import collections
+from contextlib import contextmanager
+from typing import List, Tuple
+
+import torch
+import torch.fx.traceback as fx_traceback
+
+
+# This is a list since looking forward, we can have this arbitrarily nested.
+graph_being_compiled: List[str] = []
+# TODO: It would be nice to reset the numbering every time aot_id goes
+# up, but this is annoying to do right now (because we don't know if
+# an aot_id will come back from the dead), so right now this also happens
+# to be a globally unique number too (at the cost of wobbling if you change
+# how the graphs compile)
+nth_graph: int = 0
+model_name: str = "model"
+
+
+def set_model_name(name):
```

Comment:
I see this isn't used anywhere - code-motion-only PR is great but maybe add a comment that we should just delete this later

Reply:
It's actually a public-facing API

Reply:
It's used elsewhere in torch

Reply:
yep you're right :( (used in dynamo)